stories/Teaser.tsx

```diff
@@ -1,18 +1,56 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { object, text, withKnobs } from '@storybook/addon-knobs';
 import { Teaser } from '@textkernel/oneui';
 
 storiesOf('Molecules|Teaser', module)
     .addDecorator(withKnobs)
-    .add('Teaser', () => {
+    .add('all fields', () => {
```

Comment:
For consistency with stories below, let's also add "with" `all fields` -> `With all fields` (also mind capital W)